### DISCUSSION TITLE
Phong shading?

### DISCUSSION THREAD

**Discussion by Hipshot-zz** [QodotPlugin/Qodot#138] [Q&A] [answered]
Is it possible to either define a func group or define a face with a phong value to smoothen out the looks of curved surfaces when shadow hits them inside Godot?

Edit: Smoothing Groups, similar in case someone wonders what this is or the option to Smooth in blender.

**Comment by RhapsodyInGeek** [comment, marked as answer]
You can add a `_phong` property to your brush entity and if its value is greater than 0 it will apply smooth shading to the brushes. Currently this is bugged in the sense that it only smooths on a per brush basis rather than a per entity / mesh basis. We'll be looking at resolving that with a future update.

**Comment by Hipshot-zz** [comment]
Thanks for the quick reply. Could you elaborate what you mean by bugged here? I don't fully understand your example as when I do a func group with this it seems to work, but it does look a bit strange I have to admit. The left cliff is phong 1, the right has no phong at all.

https://i.imgur.com/7lEPxiF.png

**Comment by RhapsodyInGeek** [comment]
See #101 for more information on the bug.
